src/templates/feedback.html

```diff
@@ -17,8 +17,8 @@
 {% endmacro %}
 
 <div class="main feedback">
-  <header class="secondary-header">
-    <h2 class="linefit"><span class="linefitted">Feedback</span></h2>
+  <header class="subheader">
```

Comment:
This change makes the header show up on mobile. It isn't supposed to appear until tablet width.